config/gateway.lib.yml

```diff
@@ -0,0 +1,39 @@
+#@ def gateway(system_domain, app_domains, system_namespace, workloads_namespace):
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+  name: istio-ingressgateway
+  namespace: #@ system_namespace
+spec:
+  selector:
+    istio: ingressgateway
+  servers:
+  - hosts:
+    #! TODO: restrict this
```

Comment:
Are you planning to address this TODO as part of this PR? If not, do you have a story to track this?

Reply:
@jamespollard8, @XanderStrike and I wrote [this story](https://www.pivotaltracker.com/story/show/172662780) in our tracker to figure out how we wanted to deal with this `TODO`.  We did not plan on addressing the comment in this PR.

Reply:
Ah, on first pass I didn't catch that this TODO was moved, not created

Reply:
Also, just in case it wasn't clear, the TODO wasn't added in this PR. This was in the old external-routing.yml that this was extracted from.

(I believe I added it back in the early days of cf-for-k8s when we were just racing to get anything working at all).